src/PSR7/Validators/BodyValidator/FormUrlencodedValidator.php

```diff
@@ -79,10 +79,10 @@ protected function parseUrlencodedData(MessageInterface $message) : array
     {
         $body = [];
 
-        foreach (explode('&', $message->getBody()->getContents()) as $chunk) {
-            [$name, $value] = explode('=', $chunk);
-            $body[$name]    = $value;
-        }
+        parse_str(
```

Comment:
@lezhnev74 I hope we should start support parameter serialization here

https://swagger.io/docs/specification/serialization/

required format is aligned with (in a bad way though) `deepObject` query parameters with object properties 0,1...